### ISSUE TITLE
Perform Inference with trained network on novel videos

### ISSUE THREAD

**Issue by WernTanj** [DeepLabCut/DeepLabCut#186]
I trained Resnet50 successfully with DLC on ground truth data. How can I perform inference with DLC using the best snapshot on a new, unseen video without creating a new training data set again? If I understand correctly the command 'deeplabcut.analyze_videos' requires the inputs 'config_path' and 'shuffle' which are created during the training process.

**Comment by AlexEMG** [comment]
Once you found the best shuffle (e.g. 3 and snapshot e.g. the penultimate one), then you merely have to pass these arguments on to analyze_videos or set them in the config file (depending on which one).

The **snapshot** you set in the config file, i.e. ```snaptshotindex: -2`` (if the penultimate one was the best; or e.g. 7 if it was the seventh were the order relates to the number of training steps).

Shuffle and trainingsetindex (picks out which train test split fraction) are passed on to analyze_videos (note that for making videos etc. you will need to pass the same arguments). 

```deeplabcut.analyze_videos(config, videos, **shuffle=3,** trainingsetindex=0)```

**Comment by AlexEMG** [comment]
Not sure what you mean, you can pass any folder (with videos) or list of video-paths to ```videos``` in analyze_videos and the network that is specified by the arguments will be used to analyze all those videos... 

e.g. ```deeplabcut.analyze_videos(config, ['C:\\January2019experiments\\video17.avi'], **shuffle=3,** trainingsetindex=0)```

**Comment by WernTanj** [comment]
Ok, thanks, I understand now. For subsequent analysis I am going to work always in the training folder and provide a path to videos (that were not used for training) out side of this folder.

**Comment by AlexEMG** [comment]
You can work in any folder you want (in fact we do not suggest to add videos to the config file, if they are not used for **any** trainingset). The output will be stored in the same folder as your videos. So let's assume you have trained a network and it is good (and stored somewhere as indicated by config path etc.).

Let's also say you collect '''.mp4''' videos everyday into ```D:\\myexperiments\\January\\Jan24```, ```D:\\myexperiments\\January\\Jan25``` etc. 

Then every day you run:

```deeplabcut.analyze_videos(config, ['D:\\myexperiments\\January\\Jan24'], **shuffle=3,** trainingsetindex=0,videotype='.mp4')```

then all videos from that day will be analyzed and the output added to that folder. (In fact the code is written such that if you add another video (and all others were analyzed before), then only the new video will be analyzed if you run it...)
